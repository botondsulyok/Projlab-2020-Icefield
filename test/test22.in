field spawn 0
entity master Eskimo spawn
stats spawn
stats master
item pis1 Pistol master
stats master
item cart1 Cartridge master
stats master
item ligh1 Light master
stats master
assemble master
stats spawn
stats master