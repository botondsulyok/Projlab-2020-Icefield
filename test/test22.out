id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master
pullers: master
masters: master
neighbours: null

id: master
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: master
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis1

id: master
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis1, cart1

id: master
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis1, cart1, ligh1

Game Won!
id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master
pullers: master
masters: master
neighbours: null

id: master
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis1, cart1, ligh1

